.storybook/utils.js

```diff
@@ -150,9 +150,10 @@ const renderSlots = (slots = []) =>
         typeof slot.attributes !== "undefined" &&
         Object.keys(slot.attributes).length > 0;
       if (!has_tag && (has_slot || has_attr)) {
-        Object.assign(slot, parseMarkup(slot.content));
+        let parsed = parseMarkup(slot.content);
+        Object.assign(slot, parsed);
       }
-      return slot.content
+      return slot.content || slot.tag && selfClosing.includes(slot.tag)
```

Comment:
Self-closing tags do not have "content" so we check for content *or* if a tag is defined and it's a self-closing one.